homeassistant/components/schlage/entity.py

```diff
@@ -0,0 +1,41 @@
+"""Base entity class for Schlage."""
+
+from pyschlage.lock import Lock
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import SchlageDataUpdateCoordinator
+
+
+class SchlageEntity(CoordinatorEntity[SchlageDataUpdateCoordinator]):
+    """Base Schlage entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self, coordinator: SchlageDataUpdateCoordinator, device_id: str
+    ) -> None:
+        """Initialize a Schlage entity."""
+        super().__init__(coordinator=coordinator)
+        self.device_id = device_id
+        self._attr_unique_id = device_id
```

Comment:
If you plan on adding more sensors in the future, it would be good to make the unique id `f"{devce_id}_{key}` so you don't have to have one without a key and more with

Reply:
Good call. Fixed.